### DISCUSSION TITLE
combine morot start & climate together?

### DISCUSSION THREAD

**Discussion by user7590** [TobiasLaross/Hass-Lynk-Co#44] [Q&A]
Any help in how the yaml script would be for the dashboard for having the **start engine** and **climate start** put together. so i would like to merge these 2 in one, tried many combo's but doesnt work. any help is appreciated very much:

_type: tile
entity: binary_sensor.lynk_co_vehicle_is_running
tap_action:
  action: call-service
  service: lynkco.start_engine
  target: {}
  data:
    duration_in_minutes: 15
icon_tap_action:
  action: call-service
  service: lynkco.stop_engine
  target: {}
name: 'EV Motor & Climate:'
icon: mdi:air-conditioner_

and this one:

_type: tile
entity: binary_sensor.lynk_co_pre_climate_active
tap_action:
  action: call-service
  service: lynkco.start_climate
  target: {}
  data:
    climate_level: HIGH
    duration_in_minutes: 15
icon_tap_action:
  action: call-service
  service: lynkco.stop_climate
  target: {}
vertical: false
show_entity_picture: true
name: 'Climate Control:'_

**Comment by TobiasLaross** [comment]
Create a script which you call from the front end is probably the easiest. I don’t use yaml in front end so I’m not sure how that works
